ui/package.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I needed to add this to make it work because I had installed version `4.1.x` that was incompatible.
```
"resolutions": {
    "string-width": "^4.2.0"
}
```